src/components/ProductsSelectFilter/ProductsSelectFilter.js

```diff
@@ -53,7 +53,14 @@ function ProductsSelectFilter(props) {
         >
           {mainCategory && subCategory === null && (
             <>
-              <li className={css.drop_list} onClick={clickDropList}>
+              <li
+                className={
+                  sort === null
+                    ? `${css.drop_list} ${css.active}`
+                    : css.drop_list
+                }
```

Comment:
```suggestion
                className={`${css.drop_list} ${sort === null && css.active}`}
```
이런 방법도 알고 계시면 좋을 것 같아요!

Reply:
오!! 감사합니다 진우님~~!! 훨씬 간결하고 깔끔하네요!!! 👍👍👍